resdk/tests/functional/analysis/e2e_differential_expressions.py

```diff
@@ -0,0 +1,35 @@
+# pylint: disable=missing-docstring
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from resdk.analysis import differential_expressions
+from resdk.tests.functional.base import BaseResdkFunctionalTest
+
+
+class TestExpressions(BaseResdkFunctionalTest):
+
+    def test_cuffdiff(self):
+        collection = self.res.collection.create(name='Test collection')
+
+        # pylint: disable=unbalanced-tuple-unpacking
+        cuffquant_1, cuffquant_2, cuffquant_3, cuffquant_4 = self.get_cuffquants(4, collection)
+        # pylint: enable=unbalanced-tuple-unpacking
+        gff = self.get_gtf(1)
```

Comment:
This `1` is id of collection to which gtf objects will be added. You probably don't want to do this.

Use either `self.get_gtf(collection.id)` or `self.get_gtf()`.